examples/ndjson-data-loader/rust/src/lib.rs

```diff
@@ -13,17 +13,20 @@ use std::{
     rc::Rc,
 };
 
-foxglove_data_loader::export!(NDJsonLoader);
 use foxglove_data_loader::{
     BackfillArgs, DataLoader, DataLoaderArgs, Initialization, Message, MessageIterator,
     MessageIteratorArgs, console, reader,
 };
 
+// The ID for the /accelerometer channel
+const ACC_CHANNEL_ID: u16 = 1;
+// The ID for the /temperature channel
+const TEMP_CHANNEL_ID: u16 = 2;
```

Comment:
Updated to use explicit IDs because of some data loader changes upstream. This avoids a hash lookup every message.